### PR TITLE
making ownerGroup enabled when adding a new content

### PR DESCRIPTION
**Pull request by gReis89**
it makes the Owner Groups disabled only for on edit mode:

Screen Shot 2020-08-26 at 17 27 33

Screen Shot 2020-08-26 at 17 27 57

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **greis89**

If they are not already assigned, you can assign the PR to them by writing `/assign @greis89` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/edit-content/EditContentForm.js
Comment (ichalagashvili) [inline src/ui/edit-content/EditContentForm.js:278]: `OwnerGroup` is influencing couple of attributes in the content ( I mean Content Attributes ).
That's why it was immutable in the beginning.
Since its mutable now, following Attributes should be reset/cleared/cleaned/emptied (whatever :D) when owner group changes. (Because, depending on owner group, user can choose corresponding values for those attributes).
The affected attributes are:

- AttachAttributeField
- HypertextAttributeField
- ImageAttributeField
- LinkAttributeField
- Potentially CompositeAttributeField too, as it may contain any attribute from this list (apart from Monolist)
- Potentially MonolistAttributeField too, as it also may contain any attribute from this list above (apart from Composite)

**Review by ichalagashvili — CHANGES_REQUESTED** [review]

File: src/ui/edit-content/EditContentForm.js
Comment (gReis89) [reply src/ui/edit-content/EditContentForm.js:278]: got it, thanks for the heads up!

Comment (gReis89) [reply src/ui/edit-content/EditContentForm.js:281]: @ichalagashvili I opt to use `resetSection` to clear all values from `attributes` section when changing the owner group, do you think it is good enough?

Comment (ichalagashvili) [reply src/ui/edit-content/EditContentForm.js:281]: I am not sure if UX approves it or not ( they may say that its inconvenient if other non-affected attributes will be cleared out too, but lets see). Should I approve?

Comment (gReis89) [reply src/ui/edit-content/EditContentForm.js:281]: I'll ask it in JIRA, lez hold it until we get an answer.

**Review by ichalagashvili — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-cms/pull/274#pullrequestreview-476979584" title="Approved">ichalagashvili</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)~~ [ichalagashvili]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
